docs/features/linkerd/tutorials/linkerd-authorization-policies.mdx

```diff
@@ -0,0 +1,234 @@
+---
+sidebar_position: 3
+title: Linkerd AuthorizationPolicy automation
+image: /img/quick-tutorials/k8s-linkerd-authorization-policies/social.png
+---
+
+import CodeBlock from "@theme/CodeBlock";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Otterize automates mTLS-based, HTTP-level pod-to-pod access control with Linkerd authorization (authZ) policies, within your Kubernetes cluster.
+
+Implementing this kind of access control with Linkerd can be challenging.
+For example, each target server needs a matching `Server` resource, `MeshTLSAuthentication` resources need to be created & matched with identities
+in order to use mTLS based authorization policies, etc. Using Otterize `ClientIntents`, this is all managed for you.
+
+To help you avoid manually managing complicated authorization policies per server, Otterize implements **intent-based access control** (IBAC).
+You just declare what calls the client pods intend to make,
+and everything is automatically wired together so only intended calls are allowed.
+
+In this tutorial, we will:
+
+- Deploy a Linkerd demo application with two client pods and one server pod.
+- Declare that the first client intends to call the server with a specific HTTP path and method.
+- See that a Linkerd authorization policy was autogenerated to allow just that, and to block the (undeclared) calls from the other client.
```

Comment:
What about the Server that's created, and all the other resources? We need to show those as well, and they don't exist in the Istio tutorial.